### ISSUE TITLE
Ansible 2.7 Concurrency: Further warnings...

### ISSUE THREAD

**Issue by acch** [acch/ansible-scale#4]
- [ ] tasks/scale_update.yml -- Check if any running node needs to be updated
  ```
  regression: assert msg no longer accepts list #48547 Another option to configure this is to add stdout_callback = debug to your ansible.cfg
  ```

- [x] tasks/scale_cluster.yml -- Create new cluster
  ```
  [WARNING]: flush_handlers task does not support when conditional
  ```

- [x] tasks/scale_check.yml -- Set default daemon nodename
  ```
  [WARNING]: flush_handlers task does not support when conditional
  ```

**Comment by acch** [comment]
This remains dependent on Ansible bug [#48547](https://github.com/ansible/ansible/issues/48547). To be [backported](https://github.com/ansible/ansible/pull/53311) to Ansible 2.7.

**Comment by olemyk** [comment]
The Ansible Bug #53311 is still open, This is starting to be a pain, as we can't rerun the playbooks from terraform with this bug.

**Comment by acch** [comment]
Yes, you are absolutely right. I was under the impression that the fix would be backported to Ansible 2.7. Obviously that's not going to happen any time soon - I guess we'll have to wait for Ansible 2.8...

I've re-added the workaround - sorry for the trouble this may have caused.

**Comment by acch** [comment]
Ansible bug [48547](https://github.com/ansible/ansible/issues/48547) is fixed with Ansible 2.8. Ansible 2.8 is rolled out to Fedora 29+ and RHEL 7 (via EPEL). I'm not sure about the version used with Terraform.

Do you think it's safe to revert the assert msgs to lists (remove workaround) for making them readable again?

**Comment by olemyk** [comment]
Thats good to hear, we are running Ansible 2.8 for a while now. So it should not be a problem to revert the assert msgs to list.
